spec/util/irc-client-mock.js

```diff
@@ -133,6 +133,56 @@ function Client(addr, nick, opts) {
         return this.chans[channel];
     }
 
+    // Copied over from irc.js and modified slightly
```

Comment:
Not a huge fan of this, but it looks like the IRC lib for some reason contains code that really aught to be split out. I'll allow it.

Reply:
Actually, crazy idea. Could we put these two functions in a new class in util/MessageFormatting.js or something rather than using the unsafeClient version. I think it would be better than reimplementing them in a mock.